condor_templates_test.go

```diff
@@ -108,7 +108,7 @@ func TestGenerateCondorSubmit(t *testing.T) {
 	expected := `universe = vanilla
 executable = /usr/local/bin/road-runner
 rank = mips
-requirements = (HAS_HOST_MOUNTS == True)
+requirements = (HAS_CYVERSE_ROAD_RUNNER =?= True) && (HAS_HOST_MOUNTS =?= True)
```

Comment:
Oops. Thanks for getting this after I missed it :sweat:

Reply:
No worries. I think I did the review on that PR, so we both missed it. 😄